Dockerfile

```
@@ -9,11 +9,15 @@ FROM debian:buster AS build
 # Allow users to specify forked node, or specific commit
 # If not fallback to release branch (or alternatively just specify branch)
 ARG NODE_REPO=https://github.com/casper-network/casper-node.git
-ARG NODE_COMMIT=
 ARG CLIENT_REPO=https://github.com/casper-ecosystem/casper-client-rs.git
-ARG CLIENT_COMMIT=
-ARG NODE_GITBRANCH=release-1.5.6
+ARG SIDECAR_REPO=https://github.com/casper-network/casper-sidecar.git
+
+ARG NODE_COMMIT=
+ARG NODE_GITBRANCH=feat-2.0
 ARG CLIENT_GITBRANCH=release-2.0.0
+ARG CLIENT_COMMIT=
+ARG SIDECAR_GITBRANCH=feat-2.0
```

Again, `feat-2.0` is under active development and often breaking changes are introduced. Please use this instead:

```suggestion
ARG SIDECAR_GITBRANCH=release-1.0.0rc2_node-2.0.0rc3
```

---

Fixed in 686f53fce12a6fd2e7b7590430eaf8902c22f01e.